HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  InvoicesListViewModel.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+import UIKit
+import GiniHealthSDK
+import GiniHealthAPILibrary
+import GiniCaptureSDK
+import GiniBankAPILibrary
+
+protocol InvoicesCoordinatorProtocol: AnyObject {
+}
+
+protocol InvoicesViewControllerProtocol: AnyObject {
+    func showActivityIndicator()
+    func hideActivityIndicator()
+    func reloadTableView()
+}
+
+struct DocumentWithExtractions: Codable {
+    var documentID: String
+    var amountToPay: String?
+    var paymentDueDate: String?
+    var recipient: String?
+    
+    init(documentID: String, extractionResult: GiniHealthAPILibrary.ExtractionResult) {
+        self.documentID = documentID
+        self.amountToPay = extractionResult.payment?.first?.first(where: {$0.name == "amount_to_pay"})?.value
+        self.paymentDueDate = extractionResult.extractions.first(where: {$0.name == "payment_due_date"})?.value
+        self.recipient = extractionResult.payment?.first?.first(where: {$0.name == "payment_recipient"})?.value
+    }
+    
+    init(documentID: String, extractions: [GiniBankAPILibrary.Extraction]) {
+        self.documentID = documentID
+        self.amountToPay = extractions.first(where: {$0.name == "amount_to_pay"})?.value
+        self.paymentDueDate = extractions.first(where: {$0.name == "payment_due_date"})?.value
+        self.recipient = extractions.first(where: {$0.name == "payment_recipient"})?.value
+    }
+}
+
+final class InvoicesListViewModel {
+    
+    private let coordinator: InvoicesListCoordinator
+    private weak var viewController: InvoicesViewControllerProtocol?
+    private var health: GiniHealth
+    
+    private var hardcodedDocuments: [GiniHealthAPILibrary.Document]?
+    private let dispatchGroup = DispatchGroup()
+    private let hardcodedInvoicesController: HardcodedInvoicesControllerProtocol
+    
+    var invoices: [DocumentWithExtractions] = []
+    
+    let noInvoicesText = "No Invoices"
+    let titleText = "Invoices List"
+    let uploadInvoicesText = "↑ invoices"
+    
+    let backgroundColor: UIColor = GiniColor(light: .white, dark: .black).uiColor()
+    let tableViewSeparatorCalor: UIColor = GiniColor(light: .lightGray, dark: .darkGray).uiColor()
+    
+    let tableViewCell: UITableViewCell.Type = InvoiceTableViewCell.self
+    
+    init(coordinator: InvoicesListCoordinator, viewController: InvoicesViewControllerProtocol? = nil, invoices: [DocumentWithExtractions]? = nil, giniHealth: GiniHealth, hardcodedInvoicesController: HardcodedInvoicesControllerProtocol) {
```

Comment:
[suggestion]: I think we don't need ` viewController: InvoicesViewControllerProtocol? = nil` as a parameter. 
InvoicesListCoordinator contains also the `invoicesListViewController` which is confirmed to `InvoicesViewControllerProtocol` and we can access `showActivityIndicator()` easily:
`coordinator.invoicesListViewController?.showActivityIndicator()`